v3/plugins/broadcommfd/cobol-language-support/0.13.0/meta.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v2
+publisher: broadcommfd
+name: cobol-language-support
+version: 0.13.0
+type: VS Code extension
+displayName: COBOL Language Support
+title: COBOL Language Support
+description: Autocomplete, highlighting and diagnostics for COBOL code and copybooks.
+icon: https://open-vsx.org/api/BroadcomMFD/cobol-language-support/0.13.0/file/logo.png
```

Comment:
I even don't know which option is better:
- get icon from open-vsx
or
- get the icon directly from the git repository
https://raw.githubusercontent.com/eclipse/che-che4z-lsp-for-cobol/0.13.0/clients/cobol-lsp-vscode-extension/resources/logo.png

Reply:
@vitaliy-guliy I've used open-vsx since we decided it will be our standard location for publishing extensions from now on. We can use also the icon from github repo if you'd prefer.

Reply:
I think OpenVSX or upstream GH is fine, so let's keep it this way.

Reply:
Yes, both options are working